tests/cpp/parser_tests.cpp

```diff
@@ -3521,17 +3521,17 @@ import "math"
 rule math_module
 {
 	condition:
-		math.entropy("dummy") > 7
+		math.to_number(math.entropy("dummy") > 7) == 1
```

Comment:
Should this be a brand new test instead of rewriting existing one?

Reply:
We do not have a test for each module attribute, so I personally think there is no trouble with testing more functions within one test. But I can create a new test if you would prefer it.

Reply:
Yeah, just put there more attributes.

Reply:
Done :+1: